package.json

```diff
@@ -147,9 +153,10 @@
     "embla-carousel-react": "^7.0.5",
     "envalid": "^7.3.1",
     "eth-url-parser": "^1.0.4",
-    "ethers": "^6.11.1",
+    "ethers": "6.11.1",
     "ethers5": "npm:ethers@5.7.2",
     "eventemitter2": "5.0.1",
+    "events": "^3.3.0",
```

Comment:
I don't believe this will be needed with polyfill plugin (can update to node:events import as well)

Reply:
See https://github.com/shapeshift/web/pull/8924#discussion_r1982944416